.travis.yml

```diff
@@ -44,9 +48,9 @@ deploy:
     script: make dapper-image dapper_image_flags="--nocache" release release_images="shipyard-dapper-base"
     on:
       branch: master
-      condition: $DEPLOY = true
+      condition: $RELEASE = true
```

Comment:
Good change, I was thinking recently about how this name is getting overloaded/confusing.